Refactor run.sh

carlosgalvezp opened the pull request:
Created a Python script instead for a few reasons:

-Improved readbility.
-Users more familiar with Python than bash scripts.
-Easier to maintain.
-Easier to extend.
-Platform-independent code.

J-Rojas committed:
Potential fix for bridge race condition

carlosgalvezp committed:
Refactor run.sh

Created a Python script instead for a few reasons:

-Improved readbility.
-Users more familiar with Python than bash scripts.
-Easier to maintain.
-Easier to extend.
-Platform-independent code.

carlosgalvezp committed:
Refactor run.sh

Created a Python script instead for a few reasons:

-Improved readbility.
-Users more familiar with Python than bash scripts.
-Easier to maintain.
-Easier to extend.
-Platform-independent code.

spicavigo committed:
Merge pull request #33 from Team-Autobots-SDC/feature/fix-bridge-error

Fix for bridge race condition

carlosgalvezp committed:
Use same steer_ratio in the simulator as in Carla (#49)

Finally we get the car driving around the lap!
However the car goes offroad around halfway through,
to be later investigated.

carlosgalvezp committed:
Fix yaw in wp_yaw_const.txt (#48)

Needs to be in radians, not degrees.

idanov committed:
Fix the bugs caused by too slow waypoint updater

carlosgalvezp committed:
Merge branch 'master' of https://github.com/udacity/CarND-Capstone

carlosgalvezp committed:
Merge pull request #52 from CarND-EuroBots/bugfix/speed-up-the-waypoint-updater

Fix the bugs caused by too slow waypoint updater

carlosgalvezp committed:
Unsubscribe from /base_waypoints once we got the data

This way, `waypoint_loader` will stop publishing
(since nobody subscribes to it), improving
CPU performance.

carlosgalvezp committed:
Print TL detector data to log, not screen

For consistency with other modules, and to keep the terminal
clean.

carlosgalvezp committed:
Merge pull request #55 from CarND-EuroBots/tl_output_log

Print TL detector data to log, not screen

carlosgalvezp committed:
Merge pull request #54 from CarND-EuroBots/waypoint_updater_unsubscribe

Unsubscribe from /base_waypoints once we got the data

Commit:
changes for mac

Commit:
changes for mac

orenmeiri commented:
I have made some changes to work with mac. I hope these don't brake other environments.

carlosgalvezp committed:
Use different network configuration for Linux and Mac

In Linux, it's useful to share the network in order
to use ROS tools for visualization and debugging
(rqt, rViz, etc).

carlosgalvezp committed:
Merge branch 'refactor_run_sh' of github.com:CarND-EuroBots/CarND-Capstone into refactor_run_sh
